### DISCUSSION TITLE
Can anyone else reproduce the thespruceeats.com site breakage?

### DISCUSSION THREAD

**Discussion by wavygecko** [uBlockOrigin/uAssets#14823] [Q&A]
Is anyone else able to [reproduce this site breakage](https://github.com/easylist/easylist/issues/13294)? It seems to only occur in Firefox for me. But that seems...odd?

**Comment by gorhill** [comment]
I cannot reproduce. I also cannot find any element matching `[data-ad-width]` on the page.

**Reply by wavygecko** [reply]
After I scroll down to the comments section, the site updates the `body` element attributes in my browser:

<img width="377" alt="image" src="https://user-images.githubusercontent.com/110956608/190194818-f13c9779-b36a-4bad-bb54-78cc8b1dc417.png">

**Reply by ghost** [reply]
logger content?

**Reply by wavygecko** [reply]
```yaml
uBlock Origin: 1.44.2
Firefox: 104
filterset (summary): 
  network: 88071
  cosmetic: 91843
  scriptlet: 20584
  html: 747
listset (total-discarded, last updated): 
  removed: 
    user-filters: null
  added: 
    adguard-annoyance: 59051-615, 8m
  default: 
    ublock-filters: 33432-33, 8m
    ublock-badware: 4249-1, 8m
    ublock-privacy: 267-0, 8m
    ublock-abuse: 76-0, 8m
    ublock-unbreak: 1861-0, 8m
    ublock-quick-fixes: 441-1, 8m
    easylist: 63850-624, 8m
    easyprivacy: 30575-1703, 8m
    urlhaus-1: 6907-0, 8m
    plowe-0: 3674-3, 8m
filterset (user): [array of 28 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 422 ms
  maxAssetCacheWait: 200 ms
```

**Reply by wavygecko** [reply]
Logger content:

<details><summary>Logger output</summary>


```
+0
##[id^="google_ads_iframe"]
www.thespruceeats.com
dom
https://www.thespruceeats.com/spinach-hummus-2355483
```
```
+0
##[data-ad-width]
www.thespruceeats.com
dom
https://www.thespruceeats.com/spinach-hummus-2355483
```
```
+0
##.mntl-gpt-adunit
www.thespruceeats.com
dom
https://www.thespruceeats.com/spinach-hummus-2355483
```
```
+0
###onetrust-consent-sdk
www.thespruceeats.com
dom
https://www.thespruceeats.com/spinach-hummus-2355483
```
</details>

**Reply by ghost** [reply]
Reproduced after enabling/setting Firefox's tracking protection to "Strict" or "Standard"

**Reply by wavygecko** [reply]
I think this is it! "Standard" seems to work fine for me, but "Strict" definitely causes the site breakage in my browser.

**Reply by wavygecko** [reply]
A note to my future self: If something is not working only in Firefox, there's a good chance it's related to Firefox's built-in privacy features.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/113248817/190196598-bc120733-f6d7-498b-9db9-49546964808b.png)
Reported via "Site not working?" button or whatever it is

![image](https://user-images.githubusercontent.com/113248817/190196942-68f32656-79a4-4e96-a61d-05c95729e15b.png)

**Reply by wavygecko** [reply]
Thank you!

**Reply by ghost** [reply]
You're welcome!

**Comment by ghost** [comment]
Apparently, if uBO is disabled, the issue doesn't occur.

So, if someone were to fix it, who would it be?

**Reply by gorhill** [reply]
We should create an exception in uBO's unbreak list.

**Reply by wavygecko** [reply]
When "Strict" mode in Firefox is enabled, the attribute `data-ad-width` is added to `body` which gets hidden due to https://github.com/easylist/easylist/blob/045297e8bd4d110e4bb0305e3e761da9c13fb000/easylist/easylist_general_hide.txt#L18681

**Comment by mapx-** [comment]
https://github.com/uBlockOrigin/uAssets/commit/419e378e4367cadec468b31811e03aedc3334af8

**Reply by wavygecko** [reply]
@mapx- ty! I think `.mntl-leaderboard-spacer` is an ad container so should it be in easylist instead? @ryanbr

**Reply by ghost** [reply]
@wavygecko well yes, but it appears by the allowing filter I believe

**Reply by wavygecko** [reply]
I think `thespruceeats.com##.mntl-leaderboard-spacer` is cosmetic only

whereas `thespruceeats.com#@#[data-ad-width]` fixes the breakage

**Reply by wavygecko** [reply]
but I could be wrong I've been looking at this for so long with different configs 😅
